### PR TITLE
Update v14.0.0rc + Add rust

### PR DESCRIPTION
**Pull request by oufmilo**
## PR Status

- [x] Code finished and ready to be reviewed/tested

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by oufmilo** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -99,12 +99,15 @@ ynh_add_nginx_config
 ynh_script_progression --message="Building app..."
```
Comment (lapineige) [inline scripts/upgrade]: "Installing rust and building app…" ?

Comment (oufmilo) [reply scripts/upgrade]: Yes we can change it !

Comment (oufmilo) [inline scripts/upgrade]:
```suggestion
ynh_script_progression --message="Installing rust and building app…"
```

**Pull request merged**